go/updater/update_checker.go

```diff
@@ -50,37 +61,46 @@ func (u *UpdateChecker) Check(force bool, requested bool) error {
 	return nil
 }
 
+// Start starts the update checker
 func (u *UpdateChecker) Start() {
 	if u.ticker != nil {
 		return
 	}
-	u.ticker = time.NewTicker(tickDuration())
+	u.ticker = time.NewTicker(u.tickDuration)
 	go func() {
-		for range u.ticker.C {
-			u.log.Debug("Checking for update (ticker)")
-			err := u.Check(false, false)
-			if err != nil {
-				u.log.Errorf("Error in update: %s", err)
-			}
+		for _ = range u.ticker.C {
+			go func() {
+				u.log.Debug("Checking for update (ticker)")
+				err := u.Check(false, false)
+				if err != nil {
+					u.log.Errorf("Error in update: %s", err)
+				}
+			}()
```

Comment:
The update is atomic so it can be interrupted at any moment... I might add a todo to add a test for this though